hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/helpers/TestMercifulJsonConverter.java

```diff
@@ -140,7 +140,7 @@ void decimalLogicalTypeTest(String avroFilePath, String groundTruth, String strI
     Map<String, Object> data = new HashMap<>();
 
     Schema schema = SchemaTestUtil.getSchemaFromResourceFilePath(avroFilePath);
-    GenericRecord record = new GenericData.Record(schema);
+    GenericRecord genericRecord = new GenericData.Record(schema);
```

Comment:
minor: if you're improving naming or style, let's do it in a separate PR next time. It keeps the core logic changes clean and easy to review.